src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -280,12 +292,11 @@ private function createQueryBuilder(): QueryBuilder
             ->from($this->configuration['table_name'], 'm');
     }
 
-    private function executeQuery(string $sql, array $parameters = [])
+    private function executeQuery(string $sql, array $parameters = [], array $types = [])
     {
         $stmt = null;
```

Comment:
no needed anymore

Reply:
Are you sure?
![image](https://user-images.githubusercontent.com/2140469/67408375-2af58880-f5b9-11e9-8c87-f90b73eafc63.png)

Reply:
I commented on the line `$stmt = null;`. The initialization is not needed anymore because it's always set.